packages/unleash-frontend-next/src/component/feature/AddFeatureToggle.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addFeatureToggle } from '../../action';
+
+
+let AddFeatureToggle = ({ dispatch }) => {
+    let input;
+
+    return (
+    <div>
+      <form onSubmit={e => {
+          e.preventDefault();
+          if (!input.value.trim()) {
+              return;
+          }
+          dispatch(addFeatureToggle(input.value));
```

Comment:
this is just copy-paste code from the AddTodo example.. Will not keep it long anyway